packages/block-library/src/navigation/style.scss

```diff
@@ -77,13 +77,10 @@ $navigation-icon-size: 24px;
 	}
 
 	&:where(:not([class*="has-text-decoration"])) {
-		a {
+		& :where(a),
+		& :where(a:focus),
+		& :where(a:active) {
```

Comment:
Sorry, I should have been more explicit with my suggestion in https://github.com/WordPress/gutenberg/pull/63406#discussion_r1675405046. It also took into account the suggestion made in https://github.com/WordPress/gutenberg/pull/63406#discussion_r1673862160.

The wrapping `&:where(:not([class*="has-text-decoration"]))` is not required.

Reply:
GALL@